uint8 calculation
uint8 CALCULATION_DISABLE = 49 #0x31
uint8 CALCULATION_ENABLE = 48 #0x30
---
int8 result
int8 RESULT_FAIL = -1
int8 RESULT_DONE = 0